client/src/App.js

```diff
@@ -13,16 +13,28 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
+    let newItem = true;
     const savedList = this.state.savedList;
+    savedList.map(item => {
```

Comment:
Remember that if you are using `.map` you need to return something in the method. So if you are going to do it this way then you need to use `.forEach` wit is like a normal `for loop`